src/runtime/Makefile

```diff
@@ -354,6 +380,10 @@ ifeq ($(DEFAULT_HYPERVISOR),$(HYPERVISOR_QEMU_VIRTIOFS))
     DEFAULT_HYPERVISOR_CONFIG = $(CONFIG_FILE_QEMU_VIRTIOFS)
 endif
 
+ifeq ($(DEFAULT_HYPERVISOR),$(HYPERVISOR_QEMU_NYDUS))
```

Comment:
`HYPERVISOR_QEMU_NYDUS` is not defined. Should add to here?

https://github.com/kata-containers/kata-containers/blob/df3d9878d50a83d1576cdaf5701990e0827da019/src/runtime/Makefile#L121-L130